timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -134,8 +134,8 @@ def preprocess(
     def _get_extra_tabular_init_kwargs(self) -> dict:
         raise NotImplementedError
 
-    def _get_model_params(self) -> dict:
-        model_params = super()._get_model_params().copy()
```

Comment:
Just to double check, after this PR the string `_get_model_params`  has 0 occurrences in `timeseries/`, right?

Reply:
yes 👍